pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/barnepensjon/migrering/EnkeltVedtakOmregningNyttRegelverk.kt

```diff
@@ -99,75 +103,77 @@ object EnkeltVedtakOmregningNyttRegelverk : EtterlatteTemplate<BarnepensjonOmreg
+                            "This is divided by 12 months. You will therefore receive NOK <kronebeløp> per month (pre-tax) starting 1 January 2024."
                 )
             }
 
             paragraph {
                 textExpr(
                     Language.Bokmal to "For å få full pensjon må avdødes trygdetid være beregnet til minst 40 år. Trygdetiden tilsvarer det antall år avdøde har vært medlem i folketrygden etter fylte 16 år. Når avdøde var under 67 år ved dødsfallet blir det beregnet framtidig trygdetid. Det er vanligvis fram til og med det året avdøde ville ha fylt 66 år. Barnepensjonen din er beregnet med ".expr() + anvendtTrygdetid.format() + " år trygdetid.",
                     Language.Nynorsk to "".expr(),
-                    Language.English to "".expr()
+                    Language.English to "To be entitled to a full pension, the deceased must have accumulated at least 40 years of national insurance coverage. The period of national insurance coverage equals the number of years the deceased has been a member of the Norwegian National Insurance Scheme after reaching the age of 16. If the deceased was less than 67 years old at the time of death, a calculation is made for what would have remained of the future period of national insurance coverage. This is usually calculated up to the year in which the deceased would have turned 66. Your children's pension is calculated based on a period of national insurance coverage of ".expr() + anvendtTrygdetid.format() + " years."
                 )
             }
 
             title2 {
                 text(
                     Language.Bokmal to "Skattetrekk på barnepensjon",
                     Language.Nynorsk to "",
-                    Language.English to ""
+                    Language.English to "Tax deductions on children's pensions"
                 )
             }
             paragraph {
                 text(
-                    Language.Bokmal to "Barnepensjon er skattepliktig, men vi trekker ikke skatt uten at du gir beskjed om det. Du kan lese mer om skatt på barnepensjon på nav.no/skatt-pengestotte. Her finner du også informasjon om frivillig skattetrekk og hvordan du får registrert dette.",
+                    Language.Bokmal to "Barnepensjon er skattepliktig, men vi trekker ikke skatt uten at du gir beskjed om det. Du kan lese mer om skatt på barnepensjon på ${Constants.SKATTETREKK_PENGESTOETTE_URL}. Her finner du også informasjon om frivillig skattetrekk og hvordan du får registrert dette.",
                     Language.Nynorsk to "",
-                    Language.English to ""
+                    Language.English to "Children's pensions are taxable, but we do not deduct tax without being notified. You can read more about the taxation of children's pensions online: ${Constants.SKATTETREKK_PENGESTOETTE_URL}. Here you will also find information about voluntary tax withholding and how to register this."
```

Comment:
`but we do not deduct tax without being notified` her mangler det vel `... by you`.

Reply:
Eg har ikkje så lyst til å gjera omsetjingane annleis enn det vi har fått frå kommunikasjon.